### PR TITLE
Increase Lambda timeout & memory + fix logging

### PR DESCRIPTION
**Pull request by thanhz**
Increase timeout and memory for the reference generator and fix logging

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -6,30 +6,29 @@ ThisBuild / scalaVersion := "2.13.11"
...
-  typesafe,
-  testContainer % Test,
-  testContainerDynalite % Test
-)
-
 (assembly / assemblyJarName) := "reference-generator.jar"
 
 (assembly / assemblyMergeStrategy) := {
-  case PathList("META-INF", xs @_*) => MergeStrategy.discard
+  case PathList("META-INF", "MANIFEST.MF") => MergeStrategy.discard
```
Comment (TomJKing) [inline build.sbt:32]: So I'm guessing something was missing in the JAR that was needed for logging?

File: src/main/scala/uk/gov/nationalarchives/referencegenerator/Lambda.scala [hunk 2]
```diff
@@ -57,7 +56,7 @@ class Lambda extends RequestHandler[APIGatewayProxyRequestEvent, APIGatewayProxy
 }
 
 object Lambda {
-
+  val logger: Logger = Logger[Lambda]
```
Comment (TomJKing) [inline src/main/scala/uk/gov/nationalarchives/referencegenerator/Lambda.scala:59]: Assuming the `logger` class wasn't getting constructed when the `Lambda` class was being created on invocation of the lambda?

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

File: build.sbt (hunk 1 above)
Comment (thanhz) [reply build.sbt:32]: I don't think it was the Jar I just copied this from the file-format lambda. I think the issue was the instantiating of the Logger. 😅

**Pull request merged**